clang/test/CIR/Lowering/ThroughMLIR/for-reject-1.cpp

```diff
@@ -1,9 +1,24 @@
-// RUN: not %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -fno-clangir-direct-lowering -emit-mlir=core %s -o - 2>&1 | FileCheck %s
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -fno-clangir-direct-lowering -emit-mlir=core %s -o %t.mlir
+// RUN: FileCheck --input-file=%t.mlir %s
 
-void f();
+void f() {}
 
 void reject() {
   for (int i = 0; i < 100; i++, f());
-  // CHECK: cannot handle non-constant step for induction variable
-  // CHECK: cannot handle non-canonicalized loop
+  // CHECK: %[[ALLOCA:.+]] = memref.alloca
+  // CHECK: %[[ZERO:.+]] = arith.constant 0
+  // CHECK: memref.store %[[ZERO]], %[[ALLOCA]]
+  // CHECK: %[[HUNDRED:.+]] = arith.constant 100
```

Comment:
Since these are not rejected anymore, just remove these files. Create a new one that tests the lowering for the snippet here + the one from the other test.